beluga/include/beluga/motion.hpp

```diff
@@ -15,13 +15,18 @@
 #ifndef BELUGA_MOTION_HPP
 #define BELUGA_MOTION_HPP
 
+#include <beluga/motion/ackerman_drive_model.hpp>
 #include <beluga/motion/differential_drive_model.hpp>
 #include <beluga/motion/omnidirectional_drive_model.hpp>
 #include <beluga/motion/stationary_model.hpp>
 
 /**
  * \file
  * \brief Includes all Beluga motion models.
+ *
+ * Motion models in Beluga can be classified into two categories:
```

Comment:
@fbattocchia nit: consider rephrasing this to: 
```suggestion
 * Motion models in Beluga include:
```
We may add more models in the future that don't fit these categories.